.github/workflows/functional-blockstorage.yaml

```diff
@@ -0,0 +1,62 @@
+name: functional-blockstorage
+on:
+  pull_request:
+    paths:
+      - '^.*blockstorage.*$'
+      - '.github/workflows/functional-blockstorage.yaml'
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  functional-blockstorage:
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["master"]
+        openstack_version: ["master"]
+        ubuntu_version: ["20.04"]
+        include:
+          - name: "xena"
+            openstack_version: "stable/xena"
+            ubuntu_version: "20.04"
+          - name: "wallaby"
+            openstack_version: "stable/wallaby"
+            ubuntu_version: "20.04"
+          - name: "victoria"
+            openstack_version: "stable/victoria"
+            ubuntu_version: "20.04"
+          - name: "ussuri"
+            openstack_version: "stable/ussuri"
+            ubuntu_version: "18.04"
+          - name: "train"
+            openstack_version: "stable/train"
+            ubuntu_version: "18.04"
+    runs-on: ubuntu-${{ matrix.ubuntu_version }}
+    name: Deploy OpenStack ${{ matrix.name }} with Cinder and run blockstorage acceptance tests
+    steps:
+      - name: Checkout Gophercloud
+        uses: actions/checkout@v2
+      - name: Deploy devstack
+        uses: EmilienM/devstack-action@v0.6
+        with:
+          branch: ${{ matrix.openstack_version }}
+          conf_overrides: |
+            CINDER_ISCSI_HELPER=tgtadm
+          enabled_services: 's-account,s-container,s-object,s-proxy,c-bak'
+      - name: Checkout go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15'
+      - name: Run Gophercloud acceptance tests
+        run: ./script/acceptancetest
+        env:
+          DEVSTACK_PATH: ${{ github.workspace }}/devstack
+          ACCEPTANCE_TESTS_FILTER: ^.*blockstorage(.(?!noauth).*)?$
```

Comment:
For this to work you need the same change for ACCEPTANCE_TESTS I did in https://github.com/gophercloud/gophercloud/commit/6bb11252d7cb5f6b979fa90643eb3f283244d1bc#diff-bcc2d3c788591acc07e26f74aa38c998fb12c07f2ad1bdd9ecaeea46a8705f9dR24